pkg/meta/tkv.go

```diff
@@ -2744,6 +2744,7 @@ func (m *kvMeta) dumpEntry(inode Ino, e *DumpedEntry, showProgress func(totalInc
 		a := tx.get(m.inodeKey(inode))
 		if a == nil {
 			logger.Warnf("inode %d not found", inode)
+			return nil
```

Comment:
If this is a directory, we still want all its children.

Reply:
OK, so a directory will also trigger a warning message? I see it a lot in the log

Reply:
so we can improved the case that file are delete after we saw the entry (during the dump)